CHANGELOG.md

```diff
@@ -1,278 +1,281 @@
+## 1.3.0
+
+- Add Response.badRequest() constructor
```

Comment:
I understand the desire, but please don't make unrelated changes!

Reply:
Sorry, I don't understand what you are referring to. What are the unrelated changes I made? You asked me to add this to the changelog and change the version to 1.3.0 so I did. I'm confused.

Reply:
Are you referring to the replacement of '*' with '-'? That was unintentional and was automatically done by my markdown formatter. I can certainly revert that if it makes a difference.

Reply:
Yeah, it's best. 

Send a follow-up PR for those changes. Just shouldn't mix them.

Reply:
Okay should be fixed now